web-packages/imgui-demo/src/object-inspectors/vfx-item-inspector.ts

```
@@ -65,33 +65,40 @@ export class VFXItemInspector extends ObjectInspector {
+        for (const propertyName of Object.keys(componet)) {
+          const key = propertyName as keyof Component;
           const property = componet[key];
-          const ImGuiID = componet.getInstanceId() + peopertyName;
+          const ImGuiID = componet.getInstanceId() + propertyName;
 
           if (typeof property === 'number') {
-            ImGui.Text(peopertyName);
+            ImGui.Text(propertyName);
             ImGui.SameLine(alignWidth);
             //@ts-expect-error
             ImGui.DragFloat('##DragFloat' + ImGuiID, (_ = componet[key]) => componet[key] = _, 0.03);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)